pdbtools/pdb_chainbows.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 João Pedro Rodrigues
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Renames chain identifiers sequentially, based on TER records.
```

Comment:
What about:

```
Bows chain identifiers sequentially based on TER records.

Replaces original chain identifiers.

ATTENTION: Because HETATM records are not separated by TER records and usually
come together at the end of the PDB file, this script will erroneously reassign their chain
identifiers. It is a format limitation. You have been warned!
```

If the name of the script is `chainbows` the description is preferred to be related. :wink:

Highlighted the importance of the warning

Reply:
Chainbows comes from rainbows :) I got that name from Pymol, which has a function with this name to color all chains sequentially with a rainbow gradient. I can change the name of the tool, but what would you propose that is not super complicated and unreadable (e.g. pdb_chainseqname is awful)

Reply:
What about `Rainbows chain identifiers sequentially based on TER records`? :rofl: 

Okay :-) I accept you first proposal :t-rex: